crates/meroctl/src/cli/context/create.rs

```diff
@@ -26,22 +26,37 @@ use crate::cli::RootArgs;
 use crate::common::{fetch_multiaddr, get_response, load_config, multiaddr_to_url, RequestType};
 
 #[derive(Debug, Parser)]
+#[command(about = "Create a new context")]
 pub struct CreateCommand {
-    /// The application ID to attach to the context
-    #[clap(long, short = 'a')]
+    #[clap(
+        long,
+        short = 'a',
+        help = "The application ID to attach to the context"
+    )]
```

Comment:
any reason rustdoc syntax was swapped out for `help = "__"`? imo, rustdocs are better.

Reply:
because we had a mix of both and I prefered it to be defined here as people either forget rust docs or remove them.